pkg/apis/release/v1alpha1/types.go

```diff
@@ -204,6 +207,33 @@ const (
 	ConditionPayloadRejected string = "PayloadRejected"
 )
 
+// ReleaseCreationJobResult houses the information about the Release creation batch job
+type ReleaseCreationJobResult struct {
+	// Coordinates the location of the job
+	Coordinates ReleaseCreationJobCoordinates `json:"coordinates,omitempty"`
+	// Status is the current status of the release creation job
+	Status ReleaseCreationJobStatus `json:"status,omitempty"`
+	// Message is a human readable message indicating details about the result of the release creation job
+	Message string `json:"message,omitempty"`
+}
+
+// ReleaseCreationJobCoordinates houses the information necessary to locate the job execution
+type ReleaseCreationJobCoordinates struct {
+	Name      string `json:"name,omitempty"`
+	Namespace string `json:"namespace,omitempty"`
+}
+
+type ReleaseCreationJobStatus string
+
+const (
+	// ReleaseCreationJobUnknown means the job's current state is not known
+	ReleaseCreationJobUnknown ReleaseCreationJobStatus = "Unknown"
+	// ReleaseCreationJobComplete means the job has completed its execution
+	ReleaseCreationJobComplete ReleaseCreationJobStatus = "Complete"
```

Comment:
Why Complete instead of Success?  I think Success makes more sense. You could be complete and failed.

Reply:
Done